supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesMetadataKeys.scala

```diff
@@ -5,8 +5,9 @@ object JesMetadataKeys {
   val GoogleProject = "jes:googleProject"
   val ExecutionBucket = "jes:executionBucket"
   val EndpointUrl = "jes:endpointUrl"
-  val MonitoringLog = "jes:monitoringLog"
+  val MonitoringScript = "jes:monitoringScript"
   val MachineType = "jes:machineType"
   val Zone = "jes:zone"
   val InstanceName = "jes:instanceName"
+  val MonitoringLog = "monitoringLog"
```

Comment:
I sort of equate the monitoring lot output to be kind of like a detritus output, but I'm happy to move it under the jes key.